### PR TITLE
chore: Remove all generated files

### PR DESCRIPTION
**Pull request by spydon**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -8,115 +8,106 @@ on:
...
+      - name: Restore repository cache
+        uses: actions/cache@v3
+        with:
+          path: .
+          key: ${{ runner.os }}-repo-${{ github.sha }}-generated
+          restore-keys: |
+            ${{ runner.os }}-repo-
+      - uses: Atsumi3/actions-setup-fvm@0.0.3
+      - uses: bluefireteam/melos-action@v2
+      - run: melos format:exclude
```
Comment (d-loose) [inline .github/workflows/ci.yaml:75]: It seems like the integration test went missing :)

Comment (spydon) [reply .github/workflows/ci.yaml:75]: There aren't any integration tests in this repo :smile:

Comment (d-loose) [reply .github/workflows/ci.yaml:75]: Oh wow.. why did we have that job then? :D

**Review by d-loose — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (spydon) [reply .github/workflows/ci.yaml:75]: Probably copy pasted from some other repo I'm guessing (I wouldn't be too surprised if I was the one that did it...) :sweat_smile:

**Comment by CarlosNihelton** [comment]
Honest question: how are the consumers of the `ubuntu_localizations` packge supposed to acquire the Dart-generated files when importing it from `git`?

**Comment by spydon** [comment]
> Honest question: how are the consumers of the `ubuntu_localizations` packge supposed to acquire the Dart-generated files when importing it from `git`?

They can't, if you mean adding it as a git dependency. What is the use-case for adding it as a git dependency?

**Pull request merged**